### PR TITLE
add uint8 quantized unit_test for conv2d

### PR DESCRIPTION
**Pull request by 2019dj**
Signed-off-by: Jing.Deng <Jing.Deng@verisilicon.com>

### CODE REVIEW

File: BUILD [hunk 1]
```diff
@@ -31,12 +31,15 @@ cc_library(
...
+        "src/tim/vx/test_utils.cc",
         "src/tim/transform/layout_inference.cc",
         "src/tim/transform/permute_vector.h",
         "src/tim/transform/layout_infer_context.h",
     ] + glob([
-        "src/tim/vx/ops/*.cc"
-        ], exclude = ["src/tim/vx/ops/*_test.cc"]
+        "src/tim/vx/ops/*.cc",
+        "src/tim/vx/ops/*.h"
+        ], exclude = ["src/tim/vx/ops/*test*.cc"]
```
Comment (nightingalei) [inline BUILD:42]: This change can be reverted.

File: BUILD [hunk 2]
```diff
@@ -94,7 +97,7 @@ cc_binary(
 cc_test (
     name = "unit_test",
     copts = ["-std=c++14", "-Werror"],
-    srcs = glob(["src/tim/**/*_test.cc"]),
+    srcs = glob(["src/tim/**/*test*.cc"]),
```
Comment (nightingalei) [inline BUILD:100]: This change can be reverted.

File: src/tim/vx/test_utils.cc [hunk 3]
```diff
@@ -0,0 +1,3 @@
+#include "src/tim/vx/test_utils.h"
```
Comment (nightingalei) [inline src/tim/vx/test_utils.cc:1]: Why not remove this file since it is empty?

File: BUILD [hunk 4]
```diff
@@ -31,12 +31,15 @@ cc_library(
         "src/tim/vx/tensor_private.h",
         "src/tim/vx/type_utils.h",
         "src/tim/vx/type_utils.cc",
+        "src/tim/vx/test_utils.h",
```
Comment (nightingalei) [inline BUILD:34]: These two files should be move to unit_test module, because "tim-vx_interface" doesn't need them.

Comment (sunshinemyson) [reply BUILD:34]: Since we don't put all test file in to unit test module. We have two option for this:
1. Keep test util close to where it used.
2. Move all unit_test to standalone module

Currently, I'd prefer 1st option since we don't want to move unit test to standalone moudle because we also need add unit test for private apis not only public ones.

File: BUILD [hunk 5]
```diff
@@ -35,8 +35,9 @@ cc_library(
         "src/tim/transform/permute_vector.h",
         "src/tim/transform/layout_infer_context.h",
     ] + glob([
-        "src/tim/vx/ops/*.cc"
-        ], exclude = ["src/tim/vx/ops/*_test.cc"]
+        "src/tim/vx/ops/*.cc",
+        "src/tim/vx/ops/*.h"
+        ], exclude = ["src/tim/vx/ops/*_test*.cc"]
```
Comment (thezha) [inline BUILD:40]: No need to add * after test. Please revert this change.

File: BUILD [hunk 6]
```diff
@@ -94,7 +95,9 @@ cc_binary(
 cc_test (
     name = "unit_test",
     copts = ["-std=c++14", "-Werror"],
-    srcs = glob(["src/tim/**/*_test.cc"]),
+    srcs = [
+        "src/tim/vx/test_utils.h",
+    ] + glob(["src/tim/**/*_test*.cc"]),
```
Comment (thezha) [inline BUILD:100]: No need to add * after test. Please revert this change.

**Review by thezha — CHANGES_REQUESTED** [review]
